examples/cluster.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+
+from urllib.request import Request, urlopen, HTTPError
+import json
+
+
+def request(method, url, data=None, headers=None):
+    """Makes a request against the JSON url
+    :param method: the request method
+    :param url: the path in the JSON API
+    :param data: the request body
+    :type data: bytes|str
+    """
+    req_url = url
+    req_headers = dict(headers or {})
+    req_body = None
+
+    if data is not None:
+        if isinstance(data, bytes):
+            req_body = data
+        else:
+            req_headers['Content-Type'] = 'application/json'
+            req_body = json.dumps(data).encode('utf8')
+
+    req = Request(
+        req_url,
+        req_body,
+        req_headers,
```

Comment:
Maybe remove the `req_` prefix from most of these variables? It just adds noise.